Update readme to include nectar and openssl dep (#3207)
comit-network/comit-rs: README.md

This repository is a cargo workspace:

- `cnd`: implementation of the comit-network daemon

> - `nectar`: implementation of a simple automated trader for demo purposes

Reviewer: Why just for demo purposes?

> - `comit-lib`: shared library used by `cnd` and `nectar` for swap execution

Reviewer: ```suggestion
- `comit`: shared library used by `cnd` and `nectar` that implements the COMIT protocols
```

`cnd` is [released](https://github.com/comit-network/comit-rs/releases) as binary.

## Setup build environment

All you need is ~love~ rust: `curl https://sh.rustup.rs -sSf | sh`

> For building nectar you might have to install open ssl dependencies: `sudo apt-get install libssl-dev`

Reviewer 2: ```suggestion
For building nectar you might have to install openssl dependencies: `sudo apt-get install libssl-dev`
```

Reviewer: I'd propose an alternative solution to this problem: https://github.com/comit-network/comit-rs/pull/3209
